### PR TITLE
pl/<course_name> now redirects to the PLC course enrollment page

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: dashboard/app/controllers/plc/user_course_enrollments_controller.rb [hunk 1]
```diff
@@ -3,7 +3,12 @@ class Plc::UserCourseEnrollmentsController < ApplicationController
   authorize_resource only: :create
 
   def index
-    @user_course_enrollments = @user_course_enrollments.where(user: current_user) if @user_course_enrollments
+    if params[:course]
```
Comment (aoby) [inline dashboard/app/controllers/plc/user_course_enrollments_controller.rb]: Don't you want to use the whitelisted `user_course_enrollment_params` here?

Comment (mehalshah) [inline dashboard/app/controllers/plc/user_course_enrollments_controller.rb]: Yes I do

File: dashboard/app/controllers/plc/user_course_enrollments_controller.rb [hunk 2]
```diff
@@ -3,7 +3,12 @@ class Plc::UserCourseEnrollmentsController < ApplicationController
   authorize_resource only: :create
 
   def index
-    @user_course_enrollments = @user_course_enrollments.where(user: current_user) if @user_course_enrollments
+    if user_course_enrollment_params[:course]
+      @user_course_enrollments = @user_course_enrollments.find_by(user: current_user, plc_course: Plc::Course.find_by(name: params[:course]))
```
Comment (aoby) [inline dashboard/app/controllers/plc/user_course_enrollments_controller.rb:7]: nit: `user_course_enrollment_params` here too instead of `params`

**Comment by aoby** [comment]
LGTM

**Pull request merged**